src/styles/index.scss

```diff
@@ -1,3 +1,222 @@
+@import './utils/variables';
+@import './utils/mixins';
+
+html,
+body,
+main,
+div,
+header,
+ul,
+li,
+a,
+h1,
+p,
+img {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  @include font($size: 10px, $weight: 400, $color: $main-accent-color);
```

Comment:
Verify that the mixin 'font' is correctly defined in the imported mixins file. Incorrect or missing mixin definitions can cause compilation errors.